client/blocks/nav-unification-modal/index.jsx

```diff
@@ -53,6 +53,14 @@ const Modal = () => {
 	const isDismissed = useSelector( ( state ) => getPreference( state, dismissPreference ) );
 	const translate = useTranslate();
 
+	/**
```

Comment:
Could the comment be expanded a bit to mention that "we don't extract strings from external packages, and in this case the navigation buttons' strings are coming from @wordpress/components package."?

Reply:
Good point, I've updated the comment.